src/containers/SinglePost.js

```diff
@@ -88,38 +93,43 @@ export default class SinglePost extends React.Component {
         </A>
       </div>}
       <CoverImagePage id='single-post' image={get('banner_url', community)}>
-        {editing ? <PostEditor post={post} expanded /> : showPost(post)}
-
-        {showTaggedPosts(post) && <div>
-          {currentUser && <PostEditor community={community} tag={post.tag} />}
-          <p className='meta other-posts-label'>
-            Other posts for&nbsp;
-            <span className='hashtag'>#{post.tag}</span>
-          </p>
-          {community && <ConnectedPostList subject={subject} id={community.id}
-            omit={post.id}
-            query={{...query, tag: post.tag}} />}
+        {editing
+          ? <PostEditor post={post} parentPost={parentPost} expanded />
+          : <ShowPost post={post} parentPost={parentPost} comments={comments} />
```

Comment:
I think this would read better as `showPost(post, parentPost, comments)`, since ShowPost here is different than ShowPost in PostList, and the component namespace is basically global.